Cocktails9/Cocktails9/Model/CocktailDetailViewModel.swift

```diff
@@ -0,0 +1,101 @@
+import SwiftUI
+
+class CocktailDetailViewModel: ObservableObject {
+    @Published var cocktail: Cocktail? = nil
+    @Published var isLoading = false
+    @Published var errorMessage: String? = nil
+    
+    func fetchCocktailDetails(id: String) {
+        isLoading = true
+        errorMessage = nil
+        
+        Task {
+            do {
+                let url = URL(string: "https://www.thecocktaildb.com/api/json/v1/1/lookup.php?i=\(id)")!
```

Comment:
Move this to networking class, view model should just communicate with manager and get fetched data, in this case cocktail object